pkg/global/waitgroup.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package global
+
+import "sync"
+
+var WaitGroup *sync.WaitGroup
```

Comment:
Given that it's global now, it might be handy to add a comment for `WaitGroup` explaining how producers and consumers are intended to use it

Reply:
I've added a comment noting the basic. Lmk what you think